src/app/accordion-block/accordion-block.component.scss

```
@@ -0,0 +1,14 @@
+.item-name{
+  display: flex;
+  justify-content: space-between;
+  padding: 20px;
+  border-radius: 20px;
+  border: 1px solid black;
```

use variables for colors and fonts from _variables.scss
```suggestion
  border: 1px solid $color-1;
```